applications/openshift/kubelet/kubelet_configure_event_creation_old/rule.yml

```diff
@@ -0,0 +1,71 @@
+documentation_complete: true
```

Comment:
I wouldn't be opposed to using something other than `old` in the rule name here. Primarily because "old" doesn't necessarily tell me (as a developer or user) what I should do.

What about `kubelet_configure_event_creation_deprecated`?

Reply:
that's a good point, I can change the name to *_deprecated